src/settings.js

```diff
@@ -132,6 +132,19 @@ var MALLOC = "dlmalloc";
 // how big that initial allocation (INITIAL_MEMORY) must be.
 // If you set this to 0, then you get the standard malloc behavior of
 // returning NULL (0) when it fails.
+//
+// Setting ALLOW_MEMORY_GROWTH turns this off, as in that mode we default to
+// the behavior of trying to grow and returning 0 from malloc on failure, like
+// a standard system would. However, you can still set this flag to override
+// that.
+//    * This is a mostly-backwards-compatible change. Previously this option
+//      was ignored when growth was on. The current behavior is that growth
+//      turns it off by default, so for users that never specified the flag
+//      nothing changes. But if you do specify it, it will have an effect now,
+//      which it did not previously. If you don't want that, just stop passing
+//      it in at link time.
```

Comment:
I really doubt you need to be this explicit here :)  I mean, can you imagine someone who is passing ABORTING_MALLOC reporting a bug due to malloc aborting :)   But it can't hurt.

Reply:
I agree the new behavior is more intuitive and reasonable, but still, it's a breaking change...